### PR TITLE
Feature/release notes on cli

### LINKED ISSUE #587: Add release notes and feature changelog on upgrades
**What would you like to be added**:
To see the changelog of a release upon a specific command and on upgrades

**Why is this needed**:
So the users know about all new features on Ritchie

### PR DESCRIPTION
**Pull request by victorschumacherzup**
### Description

Closes #587 
Shows the description of the latest release when command rit upgrade is used.

image

### How to verify it
Use command rit upgrade

### Changelog
Shows the description of the latest release when command rit upgrade is used.

### CODE REVIEW

**Review by henriquemoraeszup — APPROVED** [review]

File: pkg/cmd/upgrade.go [hunk 1]
```diff
@@ -94,6 +99,15 @@ func (u UpgradeCmd) runFunc() CommandRunnerFunc {
 		}
 
 		prompt.Success("Rit upgraded with success")
+		repoInfo := github.NewRepoInfo(github.RitchieRepoURL, "")
+		tag, err := u.github.LatestTag(repoInfo)
+		if err != nil || tag.Description != "" {
+			return nil
+		}
+
+		prompt.Info("Release description:")
```
Comment (kaduartur) [inline pkg/cmd/upgrade.go:108]: could you add the release version to this message?

File: pkg/git/git.go [hunk 2]
```diff
@@ -18,7 +18,8 @@ package git
 import "io"
 
 type Tag struct {
-	Name string `json:"tag_name"`
+	Name        string
```
Comment (kaduartur) [inline pkg/git/git.go:21]: as you removed the JSON keys you must put an anonymous structure in your Gitlab implementation.

**Review by kaduartur — CHANGES_REQUESTED** [review]

File: pkg/git/git.go (hunk 2 above)
Comment (victorschumacherzup) [reply pkg/git/git.go:21]: sure!

File: pkg/cmd/upgrade.go (hunk 1 above)
Comment (victorschumacherzup) [reply pkg/cmd/upgrade.go]: added

**Review by henriquemoraeszup — APPROVED** [review]

File: pkg/cmd/upgrade.go [hunk 3]
```diff
@@ -94,6 +99,21 @@ func (u UpgradeCmd) runFunc() CommandRunnerFunc {
 		}
 
 		prompt.Success("Rit upgraded with success")
+		repoInfo := github.NewRepoInfo(github.RitchieRepoURL, "")
+		tag, err := u.github.LatestTag(repoInfo)
+		if err != nil || tag.Description == "" {
+			return nil
+		}
+
+		stable, err := u.resolver.StableVersion()
```
Comment (kaduartur) [inline pkg/cmd/upgrade.go:108]: you can use `tag.Name` instead of `u.resolver.StableVersion ()`

**Review by kaduartur — CHANGES_REQUESTED** [review]

**Review by kaduartur — APPROVED** [review]
LGTM!

**Pull request merged**